### DISCUSSION TITLE
Role permissions rules

### DISCUSSION THREAD

**Discussion by massimoliani** [directus/directus#8996] [Archive]
Is there a way to enable the old role permissions rules editor?
The new one has a possible bug traversing relations.
I wrote some rules with old editor and all is working, new editor is showing it correctly.
If I try to put a new rule the editor can't allow me to traverse all the tables.

Thank you.

**Comment by benhaynes** [comment]
The old editor was just a JSON field, and you can always edit the raw JSON value by clicking on the field label context menu and selecting "raw value". This will give you what you're looking for, I believe.

**Reply by massimoliani** [reply]
Perfect thanks so much
